src/app/views/cheat-sheets/oil-refining/oil-refining.component.html

```diff
@@ -27,7 +27,7 @@ <h4 id="basic-oil">Basic Oil</h4>
 
     <p>
         When starting oil, it is beneficial to rush <a href="https://wiki.factorio.com/Advanced_oil_processing_(research)" target="_blank" rel="noopener">Advanced Oil Processing</a> research.
-        <br>This unlocks Light/Heavy Oil, and allows to obtain more efficient oil ratios by converting the Light and Heavy Oil into Petroleum Gas.
+        <br>This unlocks Light/Heavy Oil, and enables more efficient oil ratios by converting the Light and Heavy Oil into Petroleum Gas.
```

Comment:
one omore thing, dont think i use ratios appropriately here, "processes" maybe better
```suggestion
        <br>This unlocks Light/Heavy Oil, and enables more efficient oil processes by converting the Light and Heavy Oil into Petroleum Gas.
```

Reply:
Good catch. 😄  I've committed your suggestion.